README.md

```diff
@@ -1 +1,163 @@
-# ros-frontend
\ No newline at end of file
+# Resource Optimization Service (Frontend)
+
+Resource Optimization is a service that can help you optimize your public cloud workloads on Red Hat Enterprise Linux (RHEL). Read more about it [here](https://www.redhat.com/en/blog/optimize-public-cloud-workloads-rhel-red-hat-insights-resource-optimization)
+
+## Prerequisites
+
+### Install
+
+- [Node](https://nodejs.org/en/download/)
+- [Docker](https://docs.docker.com/get-docker/)
+
+
+## Setup Development Environment
+
+1. Clone below repositories:
+
+- git clone https://github.com/RedHatInsights/insights-chrome.git
+- git clone https://github.com/RedHatInsights/insights-proxy.git
+- git clone https://github.com/RedHatInsights/ros-frontend.git
+- git clone https://github.com/RedHatInsights/ros-backend.git
+
+
+Note that below file paths are based on assumption that all repositories are cloned in same folder.
+
+
+2. Make sure you meet the prerequisites mentioned in both frontend and backend repositories.
+
+3. To run the whole dev setup, you will need to run commands in multiple terminal tabs.
+
+[TAB 1] Insights Proxy TAB -
+Follow steps from [here](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/welcome/quickStart/DOC.md#proxy)
+i.e
+
+```
+cd ../insights-proxy
+npm install
+sudo bash scripts/patch-etc-hosts.sh
+bash scripts/update.sh
+
+```
+
+[TAB 2] Insights Chrome TAB -
+Wait for insights-proxy to finish update script then follow steps from [here](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/welcome/quickStart/DOC.md#chrome)
+i.e
+
+```
+cd ../insights-chrome
+npm install
+npm run build
+Open new tab [TAB 3]: ie. for ros-frontend
```

Comment:
This line is not code. Perhaps it can be a new section, since it talks of Tab 3?

Reply:
@gnurag  removed it from code and added it as separate text 👍🏻